app/views/answers/_answer.json.jbuilder

```diff
@@ -1,4 +1,5 @@
 json.extract! answer, :question_id, :comments_count, :id, :content, :votes_count, :accepted, :created_at, :updated_at
+json.voted Vote.voted(answer, current_user)
```

Comment:
Can we find a way of avoiding this query? Maybe with `eager_loading` or something? What you think @andela-oojewale, @andela-oakinniranye? :thought_balloon: